app/components/preview/components/Tabs.tsx

```diff
@@ -42,7 +47,7 @@ export function Tabs() {
             backgroundColor: cssVarStyleToken('tab.active_background'),
           }}
         >
-          <span>App.tsx</span>
+          <span>{tabs[0]}</span>
```

Comment:
Let's merge as is, but we could iterate on tabs and make all additional tabs after first have the muted appearance. Added bonus means we could revert the biome flag to make arrayIndex a warning.

Reply:
The biome warning is actually about index being used as a key in the `Indent` component when iterating over the array.